invoice/apps.py

```diff
@@ -5,21 +5,74 @@
 MODULE_NAME = 'invoice'
 
 DEFAULT_CONFIG = {
-    "default_currency_code": "USD"
+    "default_currency_code": "USD",
+    "gql_invoice_search_perms": ["155101"],
+    "gql_invoice_create_perms": ["155102"],
+    "gql_invoice_update_perms": ["155103"],
+    "gql_invoice_delete_perms": ["155104"],
+    "gql_invoice_amend_perms":  ["155109"],
+
+    "gql_invoice_payment_search_perms": ["155201"],
+    "gql_invoice_payment_create_perms": ["155202"],
+    "gql_invoice_payment_update_perms": ["155203"],
+    "gql_invoice_payment_delete_perms": ["155204"],
+    "gql_invoice_payment_refund_perms": ["155206"],
+
+    "gql_invoice_event_search_perms":             ["155201"],
```

Comment:
@dborowiecki Is this value correct? According to the wiki page - should be "155301" etc for 'EventInvoice'. https://openimis.atlassian.net/wiki/spaces/OP/pages/2858254346/Account+receivable+-+outbound+invoices

Reply:
Nice catch, fixed